### ISSUE TITLE
Android RN 0.59 Locks Screen After First Transition

### ISSUE THREAD

**Issue by jkolyer** [fram-x/FluidTransitions#177]
Using RN 0.59 on Android, the first transition works properly.  After that, the transitioned screen is non-responsive.  The navigation back button works, but the buttons in the screen do not.  This happens on each screen using the provided Examples.  

I'm trying to debug this, but wanted to notify others.

**Comment by jrwpatterson** [comment]
Do you have react-native-screens I noticed this with usescreens

**Comment by jkolyer** [comment]
Yes I just narrowed it down to some change between react-native-screens alpha.12 and alpha.22.

**Comment by jkolyer** [comment]
The problem is introduced in react-native-screens `1.0.0-alpha.18`.  alpha-17 works OK.

**Comment by jkolyer** [comment]
@chrfalch  [This change causes the blocker on Android](https://github.com/kmagiera/react-native-screens/pull/50/commits/639cf77ea1fc76dbceddddb22f834c3f06e26fce#diff-431504e69caf4f9d42d9aeb0f9ddc19a).

**Comment by chrfalch** [comment]
Could this be a problem in react-native-screens?

**Comment by jkolyer** [comment]
@chrfalch [Possibly](https://github.com/kmagiera/react-native-screens/issues/61)
